### ISSUE TITLE
The skos concepts do not belong to one ontology, they are defined like separate ontologies.

### ISSUE THREAD

**Issue by Polymnia** [julianrojas87/era-vocabulary#32]
This becomes obvious from the different prefix per class.

**Comment by julianrojas87** [comment]
Indeed, this was defined this way anticipating that the different taxonomies will be eventually published and managed independently. But for the current pilot stage they were all placed under a single ontology definition.
